TowerFox/Base.lproj/Main.storyboard

```diff
@@ -2167,7 +2167,7 @@
                                                 </constraints>
                                             </imageView>
                                         </subviews>
-                                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                        <color key="backgroundColor" name="App-textfiled-background"/>
```

Comment:
I guess, this is for the bottom button background in review screen,
do not use textfiled-background, add a new asset, and use color for dark mode as #0F0F0F
Also, fix the typo `textfiled-background`